Gruntfile.js

```diff
@@ -3,15 +3,27 @@ module.exports = function(grunt) {
 
     var pkg = grunt.file.readJSON('package.json');
 
+    /**
+     * Version Number
+     *
+     * @description
+     * Increase this number to the desired version for css and javascript files.
+     * this will also create a version directory in the css and js folders.
```

Comment:
@designerbrent Since you've changed the directories, you might want to udpate this comment.